### ISSUE TITLE
Desktop client

### ISSUE THREAD

**Issue by erlend-sh** [hummingbard/hummingbard#16]
Probably not part of any near-term plans, but might make for a good help-wanted task?

Most obvious pick for early tests is probably https://github.com/nodegui/svelte-nodegui

Later on Tauri might present an even better alternative, but it’s still a bit early: https://github.com/tauri-apps/tauri

**Comment by ChurchoftheSubGenius** [comment]
I've given this a bit of thought. Won't it be simpler to go with something like Electron? Both those projects look super cool though!

**Comment by ShadowJonathan** [comment]
If we're going with the webview route for HummingBard, then I at least recommend giving a look at [tauri](https://github.com/tauri-apps/tauri), possibly.

**Comment by erlend-sh** [comment]
True, for the first “just make it work” iteration, Electron fits the bill.

Still, the final decision is made by whoever shows up to contribute the work, so it’s good to have a few solid options.
